### DISCUSSION TITLE
Glass texture isn't correctly after upload

### DISCUSSION THREAD

**Discussion by th0rben96** [google/model-viewer#2509] [Q&A]
# Hello, 

I am new to this and I am new to Modelviewer and I want to add a .glb that is having a glass texture on top. In the Model Editor it's showing the correct black glass texture buut when I am uploading it to my server it's just white. I haven't done anything special to the code since I am new to it but I've already tried adding a Skybox without any succeess. 

Do you have any idea how to fix this issue? 

https://ibb.co/y6BN32H
https://ibb.co/WB9fdzc

So i am using the basic code: 

<!DOCTYPE html>

<head>

  </head> 

   <body contextmenu="return false" onrightclick="return false">
   
   
  		
		<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>

		
		<model-viewer src="PSQ2803MV.glb" alt="A 3D model" auto-rotate camera-controls style="width: 100%; height: 100vh; "></model-viewer>
	
    
		</body>
</html> #

**Comment by elalish** [comment]
Can you post a URL where we can see what you mean live?
